### PR TITLE
RATEST: Fix flaw in appointment legacy tests

### PR DESCRIPTION
**Pull request by kdaud**

### CODE REVIEW

File: qaframework-legacy-tests/src/test/java/org/openmrs/contrib/qaframework/legacy/AddEditAppointmentBlockTest.java [hunk 1]
```diff
@@ -46,8 +46,9 @@ public void addEditAppointmentBlockTest() throws Exception {
         appointmentBlocksPage.selectLocation(locationName);
         appointmentBlocksPage.clickOnDay();
         appointmentBlocksPage.enterStartTime(startTimeFirstAppointment);
-        appointmentBlocksPage.enterService("gyne");
```
Comment (sherrif10) [inline qaframework-legacy-tests/src/test/java/org/openmrs/contrib/qaframework/legacy/AddEditAppointmentBlockTest.java:49]: What happens when this is present.

Comment (kdaud) [reply qaframework-legacy-tests/src/test/java/org/openmrs/contrib/qaframework/legacy/AddEditAppointmentBlockTest.java:49]: "gyne" is an acronym to mean Gynecology as per the test definition. So the search indexing criteria some times is able to map the acronym to the Service type and some times fails. 

I also noticed from the UI that one has to enter Provider first before entering the service type.

Comment (sherrif10) [reply qaframework-legacy-tests/src/test/java/org/openmrs/contrib/qaframework/legacy/AddEditAppointmentBlockTest.java:49]: According to your explanation above. it looks fairly enough to be approved

**Review by sherrif10 — APPROVED** [review]
Thanks @kdaud

**Comment by kdaud** [comment]
Still looking into why ci fails the plan, will update on the subject. cc: @sherrif10

**Pull request merged**